RasimKhusaenov/my_own_blog
# Add comments

### Summary

Added comments.

### How it works

![image](https://user-images.githubusercontent.com/35406008/174796019-9e5b4b31-7c1e-4b2d-8ecf-dc595dfcb0bf.png)
![image](https://user-images.githubusercontent.com/35406008/174796107-1e0ef342-50ed-4743-ab44-d1a14d9518f9.png)

### Test plan

List of steps to manually test introduced functionality:

- Log in
- Go to https://my-own-blog-pr-62.herokuapp.com/articles/1
- Make sure you can leave a comment and it is displayed

### Deploy notes

Run `bin/rails db:migrate`

### Checklist:


### References
- Resolves #41 Issue
